src/components/SignalGeneratorComponent.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file SignalGeneratorComponent.cpp
+ * @author Luciana Moreno Borges (lumoreno.borges@gmail.com)
+ * @brief Signal Generator component
+ * @version 0.1
+ * @date 2019-01-25
+ * 
+ * @copyright Copyright (c) 2019
+ * 
+ */
+#include <hyro/SignalGeneratorComponent.h>
+#include <hyro/msgs/common/Basic.h>
+#include <hyro/SignalGenerator.h>
+#include <hyro/msgs/Signal.h>
+#include <hyro/utils/DynamicPropertyAccess.h>
+
+#include <iostream>
+
+namespace hyro
+{
+  SignalGeneratorComponent::SignalGeneratorComponent(URI uri): hyro::Component(uri){
```

Comment:
You don't need to declare an empty constructor in C++11 when you are specializing a class. In this case, we are specializing the class `hyro::Component`, therefore, you can add in the header file `using Component::Component`.

Reply:
I know, but my code doesn't work with this declaration. Do you have any idea of what it can be  ? I really tested this today.

Reply:
It should work. Do you can post the error here?